resources/deprecations.properties

```diff
@@ -168,5 +185,7 @@ vessel = https://groups.google.com/d/msg/jenkinsci-dev/L34eAMMWA5o/-AxtRdGsAAAJ
 warnings = https://issues.jenkins.io/browse/INFRA-2487
 # https://github.com/jenkinsci/xltestview-plugin/commit/ea034f9929b520e63b9ce15aed9bdb62354146cf
 xltest-plugin = https://git.io/JfaQK
+xtrigger = https://jenkins.io/security/advisory/2017-04-10/
```

Comment:
https://www.jenkins.io/blog/2017/04/10/security-advisory/#distributing-vulnerable-plugins or https://wiki.jenkins.io/display/JENKINS/Script+Security+Support+in+Plugins is probably a better reference for these plugins?